requirements.txt

```diff
@@ -0,0 +1,5 @@
+minio==5.0.4
+boto3==1.10.9
+azure==4.0.0
+azure-storage==0.36.0
+azure-functions==1.0.7
```

Comment:
Functions required? Also consider adding newline at the end of the file.

Reply:
Yes, it's true, azure-functions are not needed so I removed them from requirements (and also added a new line at the end of the file).